File: TsogosunProfileAdminWeb/src/app/model/Dtos/ms-patron-details/gmsPatronDetails.ts
export interface GMSPatronDetails {
    idpassportNumber?: string | undefined;
    countryOfIssue?: string | undefined;
    docExpiryDate?: Date | undefined;  
    nationality?: string | undefined;
    firstName?: string | undefined;
    lastName?: string | undefined;
    birthday: Date;
    gender?: string | undefined;
    address1?: string | undefined;
    address2?: string | undefined;
    pcID?: string | undefined;
    provID: number;
    countryID: number;
    pipPEP: number;
    documentType: number;
    allowCommConsent: boolean;
    allowCommSMS: boolean;
    allowCommEmail: boolean;
    allowCommPhone: boolean;
    allowCommPost: boolean;
    mobileNo?: string | undefined;
    landLineNo?: string | undefined;
    email?: string | undefined;
    photo?: string | undefined;
}

File: TsogosunProfileAdmin/MSPatronDetails/Model/Dtos/GMSPatronDetails.cs
using System;

namespace tsogosun.com.MSPatronDetails.Model.Dtos
{
    public class GMSPatronDetails
    {		
		public string IDPassportNumber { get; set; }
		public string CountryOfIssue { get; set; }
		public DateTime DocExpiryDate { get; set; } = DateTime.Now.AddYears(10);
		public string Nationality { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public DateTime Birthday { get; set; } 
		public string Gender { get; set; }
		public string Address1 { get; set; }
		public string Address2 { get; set; }
		public string PCID { get; set; }		
		public int ProvID { get; set; }
		public int CountryID { get; set; }
		public short PIPPEP { get; set; }
		public int DocumentType { get; set; }
		public bool AllowCommConsent { get; set; }
		public bool AllowCommSMS { get; set; }
		public bool AllowCommEmail { get; set; }
		public bool AllowCommPhone { get; set; }
		public bool AllowCommPost { get; set; }
		public string MobileNo { get; set; }
		public string LandLineNo { get; set; }
		public string Email { get; set; }
		public string Photo { get; set; }	

    }
}


	
	

